### PR TITLE
Workaround issue with KubernetesDeserializer

### PR DESCRIPTION
**Pull request by k-wall**

### CODE REVIEW

**Comment by k-wall** [comment]
@enmasse-ci run tests testcase=InfraTest

**Comment by k-wall** [comment]
@enmasse-ci run tests testcase=InfraTest

**Comment by k-wall** [comment]
@enmasse-ci run tests testcase=InfraTest

**Comment by k-wall** [comment]
@enmasse-ci run tests testcase=InfraTest

**Comment by k-wall** [comment]
@enmasse-ci run tests testcase=InfraTest

File: api-common/pom.xml [hunk 1]
```diff
@@ -11,7 +11,7 @@
   <dependencies>
     <dependency>
       <groupId>io.enmasse</groupId>
-      <artifactId>api-model</artifactId>
+      <artifactIdapi-model</artifactId>
```
Comment (famartinrh) [inline api-common/pom.xml]: @k-wall notice this

File: standard-controller/pom.xml [hunk 2]
```diff
@@ -11,7 +11,7 @@
   <dependencies>
     <dependency>
       <groupId>io.enmasse</groupId>
-      <artifactId>api-model</artifactId>
+      <artifactIdapi-model</artifactId>
```
Comment (famartinrh) [inline standard-controller/pom.xml]: and here too

File: api-common/pom.xml (hunk 1 above)
Comment (k-wall) [reply api-common/pom.xml]: Thanks - got it

File: standard-controller/pom.xml [hunk 3]
```diff
@@ -11,7 +11,7 @@
   <dependencies>
     <dependency>
       <groupId>io.enmasse</groupId>
-      <artifactId>api-model</artifactId>
+      <artifactId></artifactId>api-model</artifactId>
```
Comment (vbusch) [inline standard-controller/pom.xml]: artifactId is closed twice.

Comment (k-wall) [reply standard-controller/pom.xml]: Fixed -thanks.

**Comment by k-wall** [comment]
@enmasse-ci run tests testcase=InfraTest

File: api-common/pom.xml (hunk 1 above)
Comment (famartinrh) [inline api-common/pom.xml]: this is still malformed

Comment (k-wall) [reply api-common/pom.xml]: oops sorry again for the carelessness.  fixed now.

**Pull request merged**